### ISSUE TITLE
sitemap generator

### ISSUE THREAD

**Issue by ricpan** [hadronjs/hadron#18]
Any plans for a sitemap generator?

**Comment by mariocasciaro** [comment]
Yes, that's an important feature to add I think. Unfortunately I don't have much time to work on that at the moment. PRs welcome :)
